src/applications/personalization/dashboard-2/components/health-care/Prescriptions.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import HealthCareCard from './HealthCareCard';
+import { recordDashboardClick } from 'applications/personalization/dashboard/helpers';
+
+import { mhvUrl } from '~/platform/site-wide/mhv/utilities';
+
+export const Prescriptions = ({ authenticatedWithSSOe, prescriptions }) => {
+  let filteredPrescriptions;
+  let latestPrescription;
+  let nrRefillsInProgress;
+
+  if (prescriptions?.length) {
+    const thirtyDaysAgo = moment()
+      .endOf('day')
+      .subtract(30, 'days');
+    const statuses = new Set(['refillinprocess', 'submitted']);
+
+    filteredPrescriptions = prescriptions
+      .filter(p => statuses.has(p.attributes?.refillStatus))
+      .filter(
+        p =>
+          moment(p.attributes?.refillSubmitDate).isAfter(thirtyDaysAgo) ||
+          moment(p.attributes?.refillDate).isAfter(thirtyDaysAgo),
+      );
+
+    // GET LATEST DATE TO MAKE SURE LATEST PRESCRIPTION IS CORRECT
+    // This SHOULD be OK since we're filtering in loadPrescriptions
+    latestPrescription = filteredPrescriptions[0];
+    nrRefillsInProgress = filteredPrescriptions?.length;
+  }
+
+  const cardDetails = {
+    cardTitle: 'Prescription refills',
+    line1: latestPrescription?.attributes?.prescriptionName,
+    sectionTitle: 'Prescriptions',
+    line2: `Status: Submitted on ${moment(
+      latestPrescription?.attributes?.refillSubmitDate,
+    ).format('dddd, MMMM D, YYYY')}`,
+    line3: '',
+    ctaIcon: 'prescription-bottle',
+    ctaText: `${nrRefillsInProgress} prescription refill${
+      nrRefillsInProgress > 1 ? 's' : ''
+    }`,
+    ctaHref: mhvUrl(
+      authenticatedWithSSOe,
+      'web/myhealthevet/refill-prescriptions',
+    ),
+    ctaAriaLabel: 'View prescription refills',
+    ctaOnClick: recordDashboardClick('view-all-prescriptions'),
+  };
+
+  if (!nrRefillsInProgress) {
+    cardDetails.line1 = 'You have no prescription refills in progress';
+    cardDetails.cartTitle = '';
+    cardDetails.ctaText = 'Go to prescription updates';
+  }
+
+  // User has prescriptions, but none are 'refillinprocess' or 'submitted'
```

Comment:
I'm a little confused about this comment 🤔

Reply:
Some prescriptions have a `refillStatus` of `active`. If all prescriptions of a user have this status, we want to show the layout for `You have no prescription refills in progress`